proposed/http-message.md

```diff
@@ -1088,6 +1093,7 @@ interface StreamableInterface
      *     offset bytes SEEK_CUR: Set position to current location plus offset
      *     SEEK_END: Set position to end-of-stream plus offset.
      * @return bool Returns TRUE on success or FALSE on failure.
```

Comment:
I have the same question here as in rewind(): why return a boolean success flag if the failure case is throwing an exception? If we absolutely need a return value, perhaps we could return the new position in the string ala https://docs.python.org/3.1/library/io.html#io.IOBase.seek